server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -547,6 +548,8 @@ private Modal blockDescriptionModal(Tag csrfTag, BlockForm blockForm, String blo
         .withId("block-edit-form")
         .with(
             div(
+                    h3("The following fields will only be visible to administrators")
+                        .withClass("mb-2"),
```

Comment:
Note: Using the "mb-2" directly rather than `Styles.MB_2` to not cause merge conflicts with #2630, where we'll be moving to the "mb-2" pattern anyway.

Reply:
I haven't looked at 2630, but if we need to come do cleanup after that CL then do a 

//TODO(Issue #2630): xxx

Reply:
#2630 is the PR to inline Tailwind styles. The way my PR is written means there won't be any merge conflicts or need to do any cleanup after that PR. As such, I don't think there's a need for a TODO.